docs/source/index.rst

```diff
@@ -141,12 +110,12 @@ See below for how to write using Cypher via the Neo4j driver.
 
     # Upsert the vector
     vector = [random() for _ in range(DIMENSION)]
-    insert_query = (
-        "MERGE (n:Document {id: $id})"
-        "WITH n "
-        "CALL db.create.setNodeVectorProperty(n, 'vectorProperty', $vector)"
-        "RETURN n"
-    )
+    insert_query = """
```

Comment:
I'm just realizing that we have an `upsert_vector` now, shall we update this example?